File: server/src/main/java/com/backend/server/dto/DummyPaymentResponse.java
package com.backend.server.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class DummyPaymentResponse {

    private boolean success;
    private String transactionId;
    private String message;
}

File: app/src/main/java/com/example/client/data/model/DummyPaymentResponse.kt
package com.example.client.data.model

import com.google.gson.annotations.SerializedName

data class DummyPaymentResponse(
    @SerializedName("success") val success: Boolean,
    @SerializedName("transactionId") val transactionId: String? = null,
    @SerializedName("message") val message: String? = null
)
